_extensions/posit-docs/_extension.yml

```diff
@@ -0,0 +1,42 @@
+title: posit-docs
+author: Ashley Henry
+version: 0.2.0
+quarto-requred: ">=1.3.340"
+contributes:
+  project:
+    project:
+      type: website
+    website:
+      title: 
+      favicon: "assets/images/favicon.svg"
+      bread-crumbs: true
+      navbar:
+        pinned: true
+        logo: "assets/images/posit-icon-fullcolor.svg"
+        logo-alt: "Posit Documentation"  
+      sidebar:
+        style: "floating"
+        collapse-level: 1
+        search: true
+        pinned: false
+      page-footer:
+        left: 
+          - text: © 2024 Posit Software, PBC 
+            url: "https://posit.co"
+        center: |
+          Posit Product 12345 
+
+        right:
+          - icon: book
+            href: https://docs.posit.co
+          - icon: question-circle-fill
+            href: https://support.posit.co/hc/en-us
+          - icon: lightbulb-fill
+            href: https://solutions.posit.co
+  formats:
+    html:
+      theme: [cosmo, theme.scss]
```

Comment:
Should we remove cosmo from here since we don't want that added at all?

Reply:
Yep, I thought you said it was cosmo based somewhere else but I will pull that out.